tests/ndarray/test_c2array_expr.py

```diff
@@ -33,20 +33,28 @@
         # AUTH_COOKIE,
     ]
 )
-def auth_cookie(request):
-    return request.param
+def sub_auth_ctxt(request):
```

Comment:
Would it be a good idea to setup a cookie in the server for testing?

Reply:
Probably, but overhauling test machinery is out of scope of this PR.